src/app/dim-ui/Sheet.tsx

```diff
@@ -348,3 +349,77 @@ function isInside(element: HTMLElement, className: string) {
   }
   return false;
 }
+
+// After this many pixels of dragging in either direction, we consider ourselves to be part of a scrolling gesture.
+const HORIZ_SCROLL_DRAG_THRESHOLD = 20;
+
+/**
+ * We have issues on mobile where horizontal scrolling of flex components doesn't work for some unknown reason. This
+ * component is a workaround that captures pointer events when the pointer has been triggered via the down state and
+ * has also been moved by HORIZ_SCROLL_DRAG_THRESHOLD pixels. This ensures that button clicks in the component don't
+ * get interupted and work as expected.
+ */
+export function HorizontalScrollContainer({ children }: PropsWithChildren) {
```

Comment:
Could this be in a different file?

Reply:
It could be, I was debating that but since it was intended to be used with the sheet I left it there. I will move it but also rename it to `SheetHorizontalScrollContainer`

Reply:
Done